### PR TITLE
Add DefGuard  to VPN 

### PR DESCRIPTION
**Pull request by NerdvanaExplorer**
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [x] Software you are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional. Do not add a duplicate `Source code` link if it is the same as the main link. Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) for it, even if the project's webpage or readme uses another capitalisation. `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  - [DefGuard](https://defguard.net/) - True enterprise WireGuard MFA/2FA, under 250 characters, sentence case. ([Source Code](https://github.com/DefGuard), [Clients](https://defguard.net/client/)) `Apache2` `Rust`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at [awesome-selfhosted](https://awesome-selfhosted.net)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md". Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb, "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

**Please take some time to answer the following questions as best you can:**

- **Why is it awesome?**

DefGuard provides true enterprise-level security for WireGuard VPN connections with integrated multi-factor and two-factor authentication options.

- **Have you used it? For how long?**

I have been using it for a few months.

- **Is this in a personal or professional setup?**

The setup is professional.

- **How many devices/users/services/... do you manage with it?**

I use DefGuard to manage over 2000 devices and users.

- **Biggest pros/cons compared to other solutions?**

The major advantage is the enhanced security with MFA/2FA, which is not commonly offered by other VPN solutions.

- **Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

### CODE REVIEW

**Review by Technetium1 — APPROVED** [review]
This would be a good addition

File: README.md [hunk 1]
```diff
@@ -693,6 +693,7 @@ VPN software.
 - [strongSwan](https://www.strongswan.org/) - Complete IPsec implementation for Linux. ([Source Code](https://github.com/strongswan/strongswan)) `GPL-2.0` `C`
 - [tinc](https://www.tinc-vpn.org/) - Distributed p2p VPN. ([Source Code](https://www.tinc-vpn.org/git/tinc)) `GPL-2.0` `C`
 - [WireGuard](https://www.wireguard.com/) - Very fast VPN based on elliptic curve and public key crypto. ([Source Code](https://www.wireguard.com/repositories/)) `GPL-2.0` `C`
+- [DefGuard](https://defguard.net/) - True enterprise WireGuard with MFA/2FA and SSO. ([Source Code](https://github.com/DefGuard)) `License is Apache2` `Rust`
```
Comment (nodiscc) [inline README.md:696]:
```suggestion
- [DefGuard](https://defguard.net/) - True enterprise WireGuard with MFA/2FA and SSO. ([Source Code](https://github.com/DefGuard)) `Apache-2.0` `Rust`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by Technetium1 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -693,6 +693,7 @@ VPN software.
 - [strongSwan](https://www.strongswan.org/) - Complete IPsec implementation for Linux. ([Source Code](https://github.com/strongswan/strongswan)) `GPL-2.0` `C`
 - [tinc](https://www.tinc-vpn.org/) - Distributed p2p VPN. ([Source Code](https://www.tinc-vpn.org/git/tinc)) `GPL-2.0` `C`
 - [WireGuard](https://www.wireguard.com/) - Very fast VPN based on elliptic curve and public key crypto. ([Source Code](https://www.wireguard.com/repositories/)) `GPL-2.0` `C`
+- [DefGuard](https://defguard.net/) - True enterprise WireGuard with MFA/2FA and SSO. ([Source Code](https://github.com/DefGuard)) `Apache-2.0` `Rust`
```
Comment (nodiscc) [inline README.md:696]: please order entries alphabetically

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Technetium1) [reply README.md:696]: Good point

**Review by nodiscc — APPROVED** [review]

**Comment by teon** [comment]
Hi there, I'm the founder of defguard.

@NerdvanaExplorer thank you for contributing and recognizing defguard.

@nodiscc @Technetium1 - is there anything I can help/explain to push forward the merge request - would be great to spread the word.

**Pull request merged**